src/components/OverlayTooltip/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { OverlayTrigger, Tooltip } from 'patternfly-react'
+import { requiredTooltipPropType, tooltipPositionPropType } from '_/propTypeShapes'
 
-const OverlayTooltip = ({ id, tooltip, placement = 'left', children, ...rest }) => {
+const OverlayTooltip = ({ id, tooltip, placement, children, ...rest }) => {
   return (
     <OverlayTrigger
       overlay={
         <Tooltip id={id}>{tooltip}</Tooltip>
       }
-      placement={placement}
+      placement={placement || 'left'}
```

Comment:
I see that in many places we need to force position `top` - perhaps it makes sense to change the default here?
How often we need placement `left`?

Reply:
Even not once, I didn't want to change the default value to avoid unnecessary errors, but after I checked that it seems like there is no even one place in the project that using the default value, I don't know what were the considerations to choose `left` but now the best option is absolutely `top`.
So I changed it to `top`.
Thanks :)